model/infinispan/src/main/java/org/keycloak/cluster/infinispan/InfinispanNotificationsManager.java

```diff
@@ -152,30 +107,8 @@ void notify(String taskKey, Collection<? extends ClusterEvent> events, boolean i
             logger.tracef("Sending event with key %s: %s", eventKey, events);
         }
 
-        if (dcNotify == ClusterProvider.DCNotify.LOCAL_DC_ONLY || workRemoteCache == null) {
-            // Just put it to workCache, but skip notifying remoteCache
-            workCache.getAdvancedCache().withFlags(Flag.IGNORE_RETURN_VALUES, Flag.SKIP_CACHE_STORE)
-                    .put(eventKey, wrappedEvent, 120, TimeUnit.SECONDS);
-        } else {
-            // Add directly to remoteCache. Will notify remote listeners on all nodes in all DCs
-            Retry.executeWithBackoff((int iteration) -> {
-                try {
-                    DefaultInfinispanConnectionProviderFactory.runWithReadLockOnCacheManager(() ->
-                            workRemoteCache.put(eventKey, wrappedEvent, 120, TimeUnit.SECONDS)
-                    );
-                } catch (HotRodClientException re) {
-                if (logger.isDebugEnabled()) {
-                    logger.debugf(re, "Failed sending notification to remote cache '%s'. Key: '%s', iteration '%s'. Will try to retry the task",
-                            workRemoteCache.getName(), eventKey, iteration);
-                }
-
-                // Rethrow the exception. Retry will take care of handle the exception and eventually retry the operation.
-                throw re;
-            }
-
-        }, 10, 10);
-
-        }
+        workCache.getAdvancedCache().withFlags(Flag.IGNORE_RETURN_VALUES)
+                .put(eventKey, wrappedEvent, 120, TimeUnit.SECONDS);
```

Comment:
```suggestion
        CacheDecorators.ignoreReturnValues(workCache).put(eventKey, wrappedEvent, 120, TimeUnit.SECONDS);
```

Alternatively, in the constructor, do:
```java
this.workCache = CacheDecorators.ignoreReturnValues(workCache);
```